packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -59,3 +116,42 @@ export default function TemplatePartInnerBlocks( {
 
 	return <TagName { ...innerBlocksProps } />;
 }
+
+export default function TemplatePartInnerBlocks( {
+	postId: id,
+	hasInnerBlocks,
+	layout,
+	tagName: TagName,
+	blockProps,
+} ) {
+	const { canViewTemplatePart, canEditTemplatePart } = useSelect(
+		( select ) => {
+			return {
+				canViewTemplatePart:
+					select( coreStore ).canUser( 'read', 'templates' ) ?? false,
+				canEditTemplatePart:
+					select( coreStore ).canUser( 'create', 'templates' ) ??
+					false,
+			};
```

Comment:
@fabiankaegy, I've got a similar question here. Should this check if a user can read/edit this particular template part, or is it just a general check for templates?